src/components/ItemDetail/DetailSummary/SummaryListItem.js

```diff
@@ -7,7 +7,7 @@ const SummaryListItem = props => {
     <div className="summary-list-item">
       <div className="data-label">{header}</div>
       {items.map((item, index) => (
-        <div key="index" className="data-value">
+        <div key={'index_' + index} className="data-value">
```

Comment:
Is this just duplicating the `index` value separated by an `_`?   No big deal, but in general ES6 convention is `camelCase` variable names instead of `snake_case`.  When cruising code up and down the stack, this convention speeds things up I think.

Reply:
@mbklein Was this necessary?   Or related to the click firing issue?